src/wallet/wallet2.cpp

```diff
@@ -11657,6 +11667,34 @@ void wallet2::check_tx_key_helper(const crypto::hash &txid, const crypto::key_de
   }
 }
 
+bool wallet2::is_out_to_acc(const cryptonote::account_public_address &address, const crypto::public_key& out_key, const crypto::key_derivation &derivation, const std::vector<crypto::key_derivation> &additional_derivations, const size_t output_index, crypto::key_derivation &found_derivation, const crypto::view_tag *view_tag) const
```

Comment:
```suggestion
bool wallet2::is_out_to_acc(const cryptonote::account_public_address &address, const crypto::public_key& out_key, const crypto::key_derivation &derivation, const std::vector<crypto::key_derivation> &additional_derivations, const size_t output_index, crypto::key_derivation &found_derivation, const crypto::view_tag const *view_tag) const
```
A const pointer to a const view tag (i.e. document that the function won't change the pointer by marking it const).

Reply:
outdated now that I switched to boost optional type